source/Background/Keyring/index.js

```diff
@@ -136,6 +136,7 @@ export const getKeyringErrorMessage = (type) => ({
   [HANDLER_TYPES.REMOVE_NETWORK]: 'removing the network',
   [HANDLER_TYPES.SET_CURRENT_NETWORK]: 'setting the current network',
   [HANDLER_TYPES.GET_CURRENT_NETWORK]: 'getting the current network',
+  [HANDLER_TYPES.REMOVE_ASSET]: 'removing a token',
```

Comment:
adding a token is named `ADD_CUSTOM_TOKEN`. I think we should name this one `REMOVE_TOKEN` or `REMOVE_CUSTOM_TOKEN`. What u think?

Reply:
This makes perfect sense 🙏. Just updated it